server/controllers/finance/debtorGroups.js

```diff
@@ -0,0 +1,177 @@
+/**
+* The Debtor Groups Controllers
+*
+* @module finance/debtorGroups
+*
+* @desc This module is responsible for handling all crud operations relatives
+* to debtor groups, and relatives functions
+*
+* @required q
+* @required lib/db
+* @required lib/guid
+*
+* @todo implements the creation of a Debtor Group
+*/
+
+'use strict';
+
+var q  = require('q'),
+    db = require('../../lib/db'),
+    guid = require('../../lib/guid');
+
+/** Get debtor group details */
+exports.detail = getDetail;
+
+/** Get the list of debtor group */
+exports.list = getList;
+
+/** [HTTP API ENDPOINT] Get debtor groups invoices list */
+exports.getInvoices = getInvoices;
+
+/** [SERVER SIDE FUNCTION] Get debtor groups invoices list */
+exports.fetchInvoices = fetchInvoices;
+
+
+/**
+* GET /debtor_groups/:uuid
+*
+* @function getDetail
+*
+* @desc This function is responsible for retrieving details of a debtor group
+*/
+function getDetail(req, res, next) {
+  var uuid = req.params.uuid;
+
+  var debtorDetailsQuery =
+    'SELECT uuid, name, account_id, location_id, phone, email, note, locked ' +
+      'max_credit, is_convention, price_list_uuid ' +
+    'FROM debitor_group ' +
+    'WHERE uuid = ?';
+
+  db.exec(debtorDetailsQuery, [uuid])
+  .then(function (rows) {
+    if (!rows.length) { return next(new req.codes.ERR_NOT_FOUND()); }
```

Comment:
For future reference, since you are in a promise chain, this line of code is equivalent:
```js
if (!rows.length) { throw new req.codes.ERR_NOT_FOUND(); }
```